### ISSUE TITLE
New Script Request: Freeze Martingale

### ISSUE THREAD

**Issue by eckspohsdev** [currentsea/moneybot#2]
Essentially, the script needs to be configurable like a standard martingale (bet, cashout, increase on loss, etc etc).

However, I want the script to freeze / stop playing after a win for a random amount of rounds. 

Concept:

Bet: 3
Cashout: 2
Increase on loss: 2.5

On win: Math.random; < that's how many rounds to stop for (I don't know javascript very well.)
On loss: perform martingale until win

Bet Pattern for example: 

Bet 3
On loss = times by 2.5 (8 bet)
On win = random stop

Bet 8
On loss = times by 2.5 (19 bet)
On win = random stop

and so on.

Hopefully this is in enough detail for ya, currentsea. Thanks <3

**Comment by currentsea** [comment]
maxbet?

**Comment by currentsea** [comment]
@eckspohsdev can you give me an answer? <3

**Comment by eckspohsdev** [comment]
ah sorry @currentsea, haven't checked this request recently. The method I mentioned above (3 bits, 2x, 2.5) only worked due to stopping for a random amount of rounds, win or lose. I misworded my explanation - make the script stop for a random amount of rounds, even after a loss. Also, I only used that layout as an example. I initially described it as a standard martingale, meaning that users could edit the settings according to their preference, or to fit their bankroll. If you need any more help by what I mean, just post with what you need. Once again, I apologise for not checking the script request.
